setup-environment-internal

```diff
@@ -277,6 +280,9 @@ DEPLOY_DIR = "${BUILDDIR}/deploy"
 
 # Go through the Firewall
 #HTTP_PROXY = "http://${PROXYHOST}:${PROXYPORT}/"
+
+# Use public state cache mirror if no other is defined
+SSTATE_MIRRORS ??= "file://.* https://storage.googleapis.com/lmp-cache/v$LMP_VER-sstate-cache/PATH"
```

Comment:
The only issue when using site.conf is that we don't replace it when source is called (-e conf/site.conf), so developers would need to be aware of that on local development, that when a new release gets published they will have to manually erase or update the file.

The only file we updated on every source is auto.conf, maybe use that instead?

Reply:
When we have the sstate mirror in place we can delete the build folder for each build and it will be regenerated quickly. 

> The only file we update on every source is auto.conf, maybe use that instead?


However I agree that place this in conf/auto.conf will be better and will always have the version updated, so I will change it.